cli/bin/grain.js

```diff
@@ -211,6 +211,7 @@ program
 program
   .command("format [file]")
   .description("format a grain file")
+  .addOption(new ForwardOption("--in-place", "format in place"))
```

Comment:
Should be reconciled with the `.forwardOption` change in #916 (if that is merged before this PR).